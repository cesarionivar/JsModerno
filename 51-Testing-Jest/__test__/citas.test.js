import Citas from '../js/classes/Citas.js';

describe('Probar la clase de citas', ()=> {

    const citas = new Citas();

    test('Agregar una nueva cita', ()=> {

        const citaObj = {
            mascota: 'Hook',
            propietario: 'Cesario',
            telefono: '80945264120',
            fecha: '20-03-2021',
            hora: '01:28',
            sintomas: 'Solo duerme'
        };

        citaObj.id = Date.now();

        citas.agregarCita(citaObj);

        // Prueba
        expect(citas).toMatchSnapshot();

    });
});