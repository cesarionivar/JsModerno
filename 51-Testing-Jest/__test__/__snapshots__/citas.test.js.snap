// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probar la clase de citas Agregar una nueva cita 1`] = `
Citas {
  "citas": Array [
    Object {
      "fecha": "20-03-2021",
      "hora": "01:28",
      "id": 1616261677764,
      "mascota": "Hook",
      "propietario": "Cesario",
      "sintomas": "Solo duerme",
      "telefono": "80945264120",
    },
  ],
}
`;
